.github/workflows/ci-orchestrator.yml

```diff
@@ -19,19 +19,35 @@ jobs:
 
   ci_docker:
     needs: changes
-    if: ${{ needs.changes.outputs.docker == 'true' }}
     uses: ./.github/workflows/python-build-and-push-docker-image.yml
+    with:
+      docker_changed: ${{ needs.changes.outputs.docker_in_commit }}
 
   create_release:
-    needs: [changes]
+    needs: changes
     if: ${{ needs.changes.outputs.telemetry == 'true' || needs.changes.outputs.spark-sql-migrations == 'true' }}
     uses: ./.github/workflows/create-release-tag.yml
     secrets: inherit
 
+  #
+  # Run Python unit tests
+  #
+  python_ci_test:
+    needs: [ci_base, ci_docker]
+    if: always()
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci.yml@mkq/python_ci
```

Comment:
Perhaps?
```suggestion
    uses: Energinet-DataHub/.github/.github/workflows/python-ci.yml
```

Reply:
yes this is just until the .github pull request is merged :)